test/models/work_test.rb

```diff
@@ -0,0 +1,45 @@
+require "test_helper"
+
+describe Work do
+
+
```

Comment:
You didn't put your business logic in the model, so it makes sense that you wouldn't have tests for it. However, if you were to follow the comments above, then you might end up with test cases like these.

For finding the media spotlight, I would wonder:

- What happens if there are no works?
- What happens if there are works but no votes?
- What happens if two works have the same number of votes?

For finding the top ten works in a category, I would ask:

- What if there are no works of that category?
- What if there are less than 10 works?
- What if there's a tie for last place, e.g. works 9, 10 and 11 all have 0 votes?